modules/parser/src/parsers/parse-xviz-stream.js

```diff
@@ -159,7 +159,17 @@ export function parseStreamPrimitive(primitives, streamName, time, convertPrimit
   }
 
   primitiveMap.vertices = joinFeatureVerticesToTypedArrays(primitiveMap.features);
-  primitiveMap.pointCloud = joinObjectPointCloudsToTypedArrays(primitiveMap.pointCloud);
+  const pointCloud = joinObjectPointCloudsToTypedArrays(primitiveMap.pointCloud);
+  if (pointCloud) {
+    primitiveMap.features.push({
+      type: pointCloud.type,
+      points: pointCloud.positions,
+      colors: pointCloud.colors,
+      ids: pointCloud.ids
+    });
+  }
+  // Backward compatibility
+  primitiveMap.pointCloud = pointCloud;
```

Comment:
This is still referenced and used by streetscape right?  And adding the feature is to support custom layers.

Is there any way to completely remove the `primitiveMap.pointCloud` or is it not worth it?